### ISSUE TITLE
[Feature Request] Sound Loop

### ISSUE THREAD

**Issue by Deziel0495** [MGraefe/RP-Soundboard#57]
Add the ability to continuously loop sounds until they are stopped manually. A "Loop" checkbox in advanced options for each individual sound would be perfect!

**Comment by ch3mox** [comment]
yeah that's what i need for my RP to fake a Police Siren, would be nice if it will be implemented. Or is there another solution?
